Binding description

ElianaDalSasso opened the pull request:
1. Extended the description of the decoNotes available in bindingDesc with dedicated paragraphs:
- boards 
- cover
- sewing stations
- endbands
- endleaves
- spine
- slip case (leather case)
- other (parchment guards, bookmarks...)

2. Specified what can be encoded in each decoNote and which keywords are available.

3. The part concerning bookmarks is a resume of what discussed in the issue: https://github.com/BetaMasaheft/Documentation/issues/972

4. The part concerning the slip case (leather case) is still discussed in the issue:
https://github.com/BetaMasaheft/Documentation/issues/970

DenisNosnitsin1970 commented:
Dear Eliana, if i understand you right, and if you mean the description of binding arranged as
    boards
    cover
    sewing stations
    endbands
    endleaves
    spine
    slip case (leather case)
    other (parchment guards, bookmarks...)
it is fine in terms of division, though some specification would be necessary as to what exactly should be described.
Don't you want to consider separately "sewing"? we assume tacitly that bindings of Ethiopic manuscripts only have "Ethiopian link-stitch sewing", but we already have some manuscripts in the database where I would not be sure. It is good that "slip case" is included, is it the final term? Now it came to my mind that I remember at least three types of this item.  I also wonder where you would include the information on lining of the inner parts of the boards ("textile inlays").

ElianaDalSasso commented:
Dear Denis, thank you for reviewing.
Perhaps I can add a short definition of each general term (boards, cover, endbands, endleaves...) and/or add a link to the Ligatus definition. What do you think? 

In this update I specified it is possible to:
- boards: define (if known) the wood type, record the presence of textile inlay or a mirror;
- cover: describe with keywords blind-tooling decoration and record the presence of an additional leather patch;
- endbands: use the attribute color and keywords to specify the type of endband (slit-braid or link-stitch);
- endleaves: use the attribute "pastedown" in order to specify their position (for example OTI Over the Turn-Ins, UTI...);
- spine: add a free text description of special features like for example the presence of series of additional holes (perhaps related to tackets).
- other: add general remarks on the binding and/or use keywords to record 
a) the presence of parchment guards and their position;
b) the presence and the type of bookmarks.

_Sewing stations_
At the moment it is possible to record the number of sewing stations (individually, not the numbers of pairs) and the concept fits to every kind of structure (even the modern ones).
It is possible to further describe the sewing within a general decoNote with a progressive xml:id. Here it's possible to use keywords to specify the animal/vegetal origin of the thread and keywords to describe the ethiopic sewing pattern, according to G. Bozzacchi definitions.

But you are right, not always ethiopic manuscripts have ethiopic bindings, and it's interesting to record all the variants.  
Thus, your observation highlights an issue. At the moment @PietroLiuzzo made it possible to process the data encoded in bindingDesc in order to obtain graphics and statistics useful to make some observations on the Ethiopic binding.
Actually there is the risk that this data are misleading because they could incorporate values taken from bindings of other tradition (european, islamic...others). 
I don't know how is possible to keep the description related to ethiopic binding separated from others.

_Slip case_
No, there is still no agreement on the term. The issue is open (BetaMasaheft/Documentation#970) and this section may need a further update.

PietroLiuzzo commented:
Thank you @ElianaDalSasso and @DenisNosnitsin1970 
to have the charts only for a selection of manuscripts, you only need to restrict your search to that set of manuscripts you are interested in.

PietroLiuzzo commented on pages/bindingDescription.xml:
```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <gi>binding</gi>.</p>
-        </body>
+
+    <div type="level3">
```
I would add an introductory line like 'Here are some examples of how to use typed decoNote elements to provide specific information'

PietroLiuzzo replied:
see my commit

PietroLiuzzo commented on pages/bindingDescription.xml:
```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+    <div type="level3">
+      <head>Cover</head>
+       <p>In the
+             <tag>decoNote type="Cover"</tag>, using the keywords,
```
please add a link to relevant page on keywords http://betamasaheft.eu/Guidelines/?q=term&id=keywordsIntro

PietroLiuzzo replied:
see my commit

PietroLiuzzo commented on pages/bindingDescription.xml:
```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+       <p>In the
+             <tag>decoNote type="Cover"</tag>, using the keywords,
+             you can describe the the blind-tooling decoration
+             and record the presence of an additional leather patch.
```
in plain text? I do not remember what was discussed about this in the issue

PietroLiuzzo commented on pages/bindingDescription.xml:
```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+             <tag>decoNote type="SewingStations"</tag>
+             requires to type a number, which has to be equal to the number
+             of sewing stations (not to the number of pairs of sewing stations).
+         </p>
```
we definitely need an example here. what about

````xml 
<decoNote xml:id="b6" type="SewingStations">4</decoNote>
````
from SHOr405? 

see my commit

PietroLiuzzo commented on pages/bindingDescription.xml:
```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+             within a general <gi>decoNote</gi> with a progressive
+             <att>xml:id</att>. Here, if identified, you can specify with keywords the sewing pattern,
+             following Bozzacchi's classification <ref target="https://www.zotero.org/groups/358366/ethiostudies/items/itemKey/2968T6TC/q/bozzacchi"></ref>
+             and the origin of sewing thread.
```
following Bozzacchi and the origin 

or

the sawing pattern and the origin of the sawing thread ???

I am not sure, if the second is right I would anticipate perhaps

PietroLiuzzo commented on pages/bindingDescription.xml:
```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+             <tag>decoNote type="Endbands"</tag>
+             you can use the attribute <att>color</att>
+             and keywords in order to specify
+             the endband type.
```
needs example, see my commit

PietroLiuzzo commented on pages/bindingDescription.xml:
```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+             you can add the attribute <att>pastedown</att>
+             and choose up to three values prompted by the schema
+             in order to describe the position of the pastedown (if present).
+         </p>
```
needs example, see my commit

PietroLiuzzo commented on pages/bindingDescription.xml:
```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+             <tag>decoNote type="Other"</tag>
+             is possible to use keywords to record other features.</p>
+
+             <list>
```
is this list really necessary? could they not simply be paragraphs?

DenisNosnitsin1970 commented:
Eliana, here are my comments. Overall, it is well structured and thought out, and it is realistic in terms of feasibility - I mean, that a person with no deep and very special codicological background can do this. This is very important. Actually, there are some more features can be regularly recorded for each binding, but the more details are introduced, the more complex the scheme becomes. Is it your thought? I assume it is, and I refrain from including new details, except a few cases which I think are important.  Another thing - some points may be a little more difficult than the others.  

    1) boards: define (if known) the wood type, record the presence of textile inlay or a mirror; **DN: OK (but the definition of the type of the wood will not be easy; how actually to do this? we need examples of grain patterns in high resolution photos).** 
    2) cover: describe with keywords blind-tooling decoration and record the presence of an additional leather patch; **DN: OK, the description of blind-tooled decoration would be great, but it is not easy to achieve. There should be a special scheme for this.  what can be noted more easily is the presence of this decoration on the turn-ins, edges and spine. I think also colour can be recorded (I am aware that this is very arbitrary and difficult, but we can try), and if the leather cover is full cover or part cover ("quarter"). The last feature is important for Ethiopic manuscripts, I think there is no appropriate terminology ("quarter cover" is not exact)**  
   3)  endbands: use the attribute color and keywords to specify the type of endband (slit-braid or link-stitch); OK
    4) endleaves: use the attribute "pastedown" in order to specify their position (for example OTI Over the Turn-Ins, UTI...); **DN: is it more common to consider endleaves as part of the textblock or part of the binding? I wonder if it is appropriate to tacitely assign them to "binding". There will be need of a definition and some examples in any case, because in many cases it is difficult to understand, if the leaves are endleaves or not.  "Pastedown" means for me something different; in my understanding, it is very important, though, if one can really understood if the volume has or had "pastedowns". By the way, I would go another way around: I would make stress on describing "turn-ins", developing from "cover". There can be several clear definitions applied to them.**
    5) spine: add a free text description of special features like for example the presence of series of additional holes (perhaps related to tackets). "DN: OK, I used to record, if fitting, "concave", "convex". have you seen any volume with "lined spine" (Islamic manuscripts)?
   6) fastening (missing in the list above): record their presence or trace of them; **DN: OK**
    other: add general remarks on the binding and/or use keywords to record
    7) a) the presence of parchment guards and their position; **DN: OK, but it is in the bookblock, right?**
    b) the presence and the type of bookmarks. **DN: OK, we discussed this**

Sewing stations
At the moment it is possible to record the number of sewing stations (individually, not the numbers of pairs) and the concept fits to every kind of structure (even the modern ones). **DN: do you propose to number them 1 - 2 - 3 - 4, starting from the head? Why not mention pairs where applicable?**
It is possible to further describe the sewing within a general decoNote with a progressive xml:id. Here it's possible to use keywords to specify the animal/vegetal origin of the thread and keywords to describe the ethiopic sewing pattern, according to G. Bozzacchi definitions. DN: that is fine, but then G. Bozacchi's definitions should be presented in English, in consize and clear way, again with examples. 
DN

ElianaDalSasso commented:
Hi, not yet, I am modifying some things according to your and Denis suggestions.

ElianaDalSasso commented:
Hallo everyone and happy new year!
Thank you for the punctual review, I agree with @DenisNosnitsin1970, there are still a lot of features for each element that can be recorded. This list is a resume of the main features I encountered during my MA project and I would be happy to expand it. Of course it depends also on the interest and time of the cataloguers. 

1) Bookblock/Bookbinding
A first issue seems to be how to place correctly the description of the different features. 
_Endleaves_ and the _parchment_ _guards_ are actually part of the bookblock. 
Would it be more appropriate to describe them, maybe, in physDesc>objectDesc>supportDesc>support ? @PietroLiuzzo what do you think?

I think it would raise again the problem of keeping or not leather case and bookmarks in bookbinding description.

2) Endleaves and pastedowns
What do you think of this description of the two terms? (see commit)
"Endleaves are leaves intended to protect the text leaves. Therefore they are placed at the beginning and/or at the end of the text. Endleaves can be extra leaves added by the binder during the sewing or they can be leaves belonging to the first or last quire, left blank by the scribe in order to protect the text. Later, endleaves can be filled with notes, additional texts, ecc. 
If an endleaf is pasted to the inside of a board it's called pastedown and it is important to record its position in relation with the turn-ins. 
Inside the <tag>decoNote type="Endleaves"</tag> you can add the attribute <att>pastedown</att>
and choose up to three values prompted by the schema in order to describe the position of the pastedown. See for example <ref type="BM" target="SHOr405">Hamburg, State and University Library Hamburg Carl von Ossietzky, Cod. Orient. 405</ref>: <egXML xmlns="http://www.tei-c.org/ns/Examples"><decoNote xml:id="b11" type="EndLeaves" pastedown="L OTI">
The inner surface of the front board is covered with paper. Traces of paper are found also on the external surface."

3) Turn-ins
At the moment there are no keywords available to describe turn-ins features, but it would be very interesting to have them. I encountered some overlapping head and tail turn-ins, mainly in quarter cover. I know of "sewn turn-ins" but Ligatus offers a wide range of other examples. 
@DenisNosnitsin1970 do you suggest to add some keywords to describe particular aspects of turn-ins?Their description could fit into the cover description.

4) Cover
- Decoration
I tried to resume the main decoration motives found in literature and organise them in a simple schema in Word (.doc), I am going to forward it to you @DenisNosnitsin1970. I think it's possible to name the principal motives. 
It would be clearer using images but I don't think is possible to add images from articles.

Sure it's possibile to specify the placement of the decoration (see commit). 
@PietroLiuzzo is there a way to encode this? I was thinking to use for example `<locus target="#b9"/>`  where b9=spine but we don't have an item for edges and turn-ins.

- Color
I don't know how we could record it, would it be in free text or with keywords?Then we need a selection of shades of colours.

- Quarter cover
Quarter cover is a common term to refer to covers that cover only the spine and the back edge of the boards even if it extends till the half or more. If you agree I can open an issue and suggest to add a keyword "quarter cover" and then add it in guidelines.

5) Boards 
The keywords available are a collection of all the type of woods said to be used in ethiopic bookbinding that I found in literature. The identification of a specie is not simple and needs a specialist. A cataloguer could record the feature if it is specified in the catalogue or in articles related to that manuscript.

In the MA project at the university, the scientist and I took very small and thin layers of wood from the boards and observed them through optical and stereoscopic microscope. The identification it's not easy because it requires a high experience to recognize pattern of fibers, vessels...finally we menage to identify a Cordia and a Ficus type.
I was discussing that with @PietroLiuzzo if it worth to keep this part because without a sound research project behind (that involve taking samples from the original) actually it's not possible for a cataloguer to identify the wood. 

6) Spine
@DenisNosnitsin1970 what do you mean for concave and convex?If it is concave I think it is a damage of the structure, that can be caused by a very tight sewing. A convex spine is defined as "round" (or slightly round, semi-elliptical) but for traditional ethiopic binding I would say it's not made on purpose and can be a damage of the structure. Maybe I misunderstood your observation.
Yes, I have seen volumes with lined spine but they were not ethiopic binding. What I have seen in few ethiopic is that the head and tail turn-ins are very wide so they can form a kind of second layer along the spine. Another type is like Hamburg Or. 410 a double leather layer. First layer is used to sew the endbands (so the stitches are not seen from outside) and the second is a quarter cover. But it was a later repair.

7) Sewing stations
Yes I agree in numbering from head to tail (1,2,3..). It is better to encode the number of sewing stations because when there are three sewing stations the pairs are always two (the second sewing station is shared). 
In betamasaheft app in resources>keywords there is a description in english for every sewing pattern. for example: http://betamasaheft.eu/authority-files/list?keyword=patternA
Could it work or should it be specified in guidelines?

PietroLiuzzo commented:
I think we could go ahead with the current version and keep this questions as they are nicely documented for a later time.

PietroLiuzzo commented:
no worries, you do not need me to merge, please just go ahead when you are done.

DenisNosnitsin1970 commented:
Dear Eliana, here are my comments.
1.	Bookblock/Bookbinding
A first issue seems to be how to place correctly the description of the different features.
Endleaves and the parchment guards are actually part of the bookblock.
Would it be more appropriate to describe them, maybe, in physDesc>objectDesc>supportDesc>support ? @PietroLiuzzo what do you think?
I think it would raise again the problem of keeping or not leather case and bookmarks in bookbinding description. **DN: I personally find keeping leather case in bookbinding description fine; in a way, leather case has de facto a similar function. By the way, we can have a look in other catalogues (not only of Ethiopic mss) to see how it is treated.**
2.	Endleaves and pastedowns
What do you think of this description of the two terms? (see commit)
"Endleaves are leaves intended to protect the text leaves. Therefore they are placed at the beginning and/or at the end of the text. Endleaves can be extra leaves added by the binder during the sewing or they can be leaves belonging to the first or last quire, left blank by the scribe in order to protect the text. Later, endleaves can be filled with notes, additional texts, ecc. **DN: “endleaves” for me are essentially the “extra leaves”, in smaller quires which are meant for protecting the textblock. They are left blank, exactly, and frequently are neither pricked and nor ruled. Another way – application of leaves used as “maculature”.  For me, this is their functional distinction – they are not meant for writing, but for something else (can be used, of course, for marginal writings). In Ethio-SPaRe we used to count them separately, A and B. From my experience, not too many manuscripts have clearly identifiable endleaves, and a still lesser number have them on both sides of the textblock. Of course, they are unstable elements of the book. I would not really tend to describe as endleaves simply a leaf of the first text quire left blank, and the last leaf of the last text quire left blank either. For me they would be just blank leaves, but not “endleaves”. This is just my opinion**
3.	  
If an endleaf is pasted to the inside of a board it's called pastedown and it is important to record its position in relation with the turn-ins.
Inside the decoNote type="Endleaves" you can add the attribute pastedown
and choose up to three values prompted by the schema in order to describe the position of the pastedown. See for example Hamburg, State and University Library Hamburg Carl von Ossietzky, Cod. Orient. 405: 
The inner surface of the front board is covered with paper. Traces of paper are found also on the external surface."
4.	Turn-ins
At the moment there are no keywords available to describe turn-ins features, but it would be very interesting to have them. I encountered some overlapping head and tail turn-ins, mainly in quarter cover. I know of "sewn turn-ins" but Ligatus offers a wide range of other examples.
@DenisNosnitsin1970 do you suggest to add some keywords to describe particular aspects of turn-ins?Their description could fit into the cover description.
**DN: In the codicological destriptions of Ethio-Spare we used the following terms: mitered corners (frequently spelled “mitred”), open mitered corners, fore edge over, head/tail over, tongues (these terms were in the extended codicological description form which was elaborated by Marco Di Bella and Nikolas Sarris, ultimately also basing on their Ligatus experience).  “Mitered” is the most frequent, but also another type, of stitched turn ins, occurred to me.**  
5.	Cover
•	Decoration
I tried to resume the main decoration motives found in literature and organise them in a simple schema in Word (.doc), I am going to forward it to you @DenisNosnitsin1970. I think it's possible to name the principal motives.
It would be clearer using images but I don't think is possible to add images from articles.
Sure it's possibile to specify the placement of the decoration (see commit).
@PietroLiuzzo is there a way to encode this? I was thinking to use for example <locus target="#b9"/> where b9=spine but we don't have an item for edges and turn-ins.
•	Color
I don't know how we could record it, would it be in free text or with keywords?Then we need a selection of shades of colours. **DN: We also found that defining the colour of anything in the image is a true challenge. On the other hand, it is very odd to abstain from it completely. We tried to follow calibrated color tables, such as here, www.procato.com/color+weel/?hue=330; there are also other similar devices where you get a good idea of the color and the term, but I felt that someone should work a bit more on this, and see if it really works out in descriptions. Of course, one should not try to be too precise as it is simply impossible.**
•	Quarter cover
Quarter cover is a common term to refer to covers that cover only the spine and the back edge of the boards even if it extends till the half or more. If you agree I can open an issue and suggest to add a keyword "quarter cover" and then add it in guidelines. **DN: yes, I agree.** 
5.	Boards
The keywords available are a collection of all the type of woods said to be used in ethiopic bookbinding that I found in literature. The identification of a specie is not simple and needs a specialist. A cataloguer could record the feature if it is specified in the catalogue or in articles related to that manuscript.
In the MA project at the university, the scientist and I took very small and thin layers of wood from the boards and observed them through optical and stereoscopic microscope. The identification it's not easy because it requires a high experience to recognize pattern of fibers, vessels...finally we menage to identify a Cordia and a Ficus type.
I was discussing that with @PietroLiuzzo if it worth to keep this part because without a sound research project behind (that involve taking samples from the original) actually it's not possible for a cataloguer to identify the wood.
**DN: yes, I agree. It would be very desirable, but it is hardly possible, at least at this stage.** 
6.	Spine
@DenisNosnitsin1970 what do you mean for concave and convex?If it is concave I think it is a damage of the structure, that can be caused by a very tight sewing. A convex spine is defined as "round" (or slightly round, semi-elliptical) but for traditional ethiopic binding I would say it's not made on purpose and can be a damage of the structure. Maybe I misunderstood your observation. **DN: you understand  right, convex is “raised”, concave is “sunken”, and there is “the normal one”. Concave/convex is not made on purpose, but it is not necessarily mechanical damage, yet rather a gradual deformation of the textblock as a consequence of badly constructed binding (thick textblock, too thin/light borders; I saw this feature on bindings with leather boards). On my part, I would somehow notice the presence, if it is there, but it must not be, though. It is not very frequent.**   
Yes, I have seen volumes with lined spine but they were not ethiopic binding. What I have seen in few ethiopic is that the head and tail turn-ins are very wide so they can form a kind of second layer along the spine. Another type is like Hamburg Or. 410 a double leather layer. First layer is used to sew the endbands (so the stitches are not seen from outside) and the second is a quarter cover. But it was a later repair. **DN: not sure that I understand you fully; do you mean a kind of tongue, “tab” of the leather cover, is folded upon the spine, so that the endband is sewn to the double leather layer? I think it is the normal structure, somewhat difficult to observe (mostly on damaged spines). If there is no true lining found so far, maybe we can just skip it
![turn ins](https://user-images.githubusercontent.com/18533839/51051366-e2458380-15d3-11e9-9144-7eb2bcbb547b.png)
.**    
7.	Sewing stations
Yes I agree in numbering from head to tail (1,2,3..). It is better to encode the number of sewing stations because when there are three sewing stations the pairs are always two (the second sewing station is shared).
In betamasaheft app in resources>keywords there is a description in english for every sewing pattern. for example: http://betamasaheft.eu/authority-files/list?keyword=patternA
Could it work or should it be specified in guidelines?
**DN: the descriptions of the sewing patterns might be correct, but I find them quite difficult, frankly speaking. Each describes the process which is essentially the same but has subtle variations, difficult to grasp in an abstract form, even for me, and probably much more for the cataloguers. Also, the patterns are quite many, over 10, so it is very detailed (I assume some are more and some are less used). I am sorry, I do not know how I would learn to distinguish between them, unless the sewing of a manuscript is very well-preserved and I have access it, or to the images where it is clearly visible, and I have “models” in front of my eyes. Could you think if it can be re-arranged or presented in a different way, and made better understandable? (In A, are four threads used for two pairs of sewing stations, or two double threads?).**

ElianaDalSasso commented:
Thank you @DenisNosnitsin1970 

1. Bookblock/bookbinding
Ok but I don't think that these features (endleaves, parchment guards, leather case, bookmarks) are often recorded in catalogues. 
According to TEI guidelines these elements have to be described in `<physDesc>`. 
They don't fit in the group which concerns aspects of the writing, illumination, or other notation.
In my opinion they can't be described in `<objectDesc>` because this element is used to group together those parts of the physical description which relate specifically to the text-bearing object. But with an elastic interpretation endleaves, pastedown and guards could be added to `<support>` or to `<collation>`.
Maybe the easiest solution is to keep the sections as they are, in binding description (paragraph 10.7.3 "Bindings, Seals, and Additional Material" of TEI guidelines). Thus, bookmarks, leather case and guards could eventually be moved in an element `<accMat>` which 

> contains details of any significant additional material which may be closely associated with the manuscript being described, such as non-contemporaneous documents or fragments bound in with the manuscript at some earlier historical period.

2. Endleaves and pastedowns
Yes, I agree that in ethiopic manuscript endleaves are not frequent and most of the time they are extra leaves. But "endleaves" is a general term that describes an element of the bookstructure and by definition they can be integral (part of a text quire) or separate (extra leaves). 
I think that what is important for the schema is to have the possibility to record the relation between pastedown and turn-ins. 
What do you think about this formula:
`Endleaves are leaves intended to protect the text leaves. Therefore they are placed at the beginning and/or at the end of the text. Endleaves are left blank but later they can be used for marginal writings.
           If an endleaf is pasted to the inside of a board it's called pastedown and it is important to record its position in relation with the turn-ins.
           Inside the <tag>decoNote type="Endleaves"</tag> you can add the attribute <att>pastedown</att>
           and choose up to three values prompted by the schema in order to describe the position of the pastedown.
           See for example <ref type="BM" target="SHOr405">Hamburg, State and University Library Hamburg Carl von Ossietzky, Cod. Orient. 405</ref>:
           <egXML xmlns="http://www.tei-c.org/ns/Examples">
           <decoNote xml:id="b11" type="EndLeaves" pastedown="L OTI">
           The inner surface of the front board is covered with paper. Traces of paper are found also on the external surface.`

3. Turn-ins
Ok, we can collect those terms and open an issue to create keywords to use in cover description.

4. Cover
I agree with you with the fact that a precise description of color is impossible at glance. In fact color perception changes according to the illumination and the system which records it.
It would be possible using scientific measures (colorimeters, spectrometers...). At the moment I don't think it's possible to produce a list of reliable colours. 
However the schema gives the opportunity to add the attribute "color" where the cataloguer can specify (arbitrary) the color of the cover. 
What do you think about this:
`In the
             <tag>decoNote type="Cover"</tag> you can describe the covering of the binding.
             <!--If it covers only the spine and the back edge of the boards use the <att>key</att> quarterCover to encode it.-->
             The attribute <att>color</att> is available
             and using <ref target="keywordsIntro">keywords</ref>,
             you can record the presence of an additional leather patch and describe the the blind-tooling decoration.
             It is useful to specify where the decoration is found (boards, edges, turn-ins, spine).
             <!--Additional keywords are available to describe the aspect of the turn-ins.-->`

5. Spine
I would us the terms concave and convex in the element <condition> to mark it's something happened later. About the lining yes I agree in skipping it. (the example was a particular structure, originated from a repair)

7. Yes, the sewing pattern are a very specific topic and it is very difficult to grasp the differences between them without building, sewing models. I can try to rearrange the description classifying them according to what you see in the centre of the first, last quires and on the boards. 
(In A there are 4 threads, one for each sewing station (i.e. 4 threads for two pairs of sewing stations))

DenisNosnitsin1970 commented:
Dear Eliana, a few more remarks: 

1) OK, agree. We could try to incorporate pastedowns and guards into „collation“ and visualize them, this would be interesting. Though, of course, an additional complicated task. 
2) Then you would need further specifications in this category. Both endleaves types are different in terms of technology. From my experience, the “integrated endleaves" are not frequent and this is not the dominant practice (also, I would not remember any example immediately). But if you mean it occurs, a means for describing it can be designed, sure. I would say: 
“Endleaves: one or more leaves added at the front and/or back of the textblock for its protection. Endleaves are frequently left blank or without pricking and ruling; but they can be used for marginal writing.“ 
“Pastedown: an endleaf adhered to the inside of the front of back board. It is important to record the position of the pastedown in relation with the turn-ins.”
I would just distinguish the definitions for the endleaves and pastedowns. They are related, but are treated as separate terms in all glossaries. 
One general question is: should this terminology be applicable to the Ethiopic manuscripts only, or be more inclusive (I assume the second, as we have also Islamic manuscripts)?

PietroLiuzzo commented:
Could this now be merged?

ElianaDalSasso commented:
Dear @DenisNosnitsin1970 and @PietroLiuzzo I updated the binding description according to what said in the last comments. In particular I modified the definition of endleaf/pastedown as suggested by Denis and I removed the part concerning the sewing pattern (it can always be added later since it's ready but it's true, it needs to be simplified in order to be widely used).
If you agree with the commits it can be merged.
